lib/c-api/src/wasm_c_api/wasi/mod.rs

```diff
@@ -409,12 +501,6 @@ pub unsafe extern "C" fn wasi_get_start_function(
     }))
 }
 
-/// Delete a `wasm_extern_t` allocated by the API.
-///
-/// cbindgen:ignore
-#[no_mangle]
-pub unsafe extern "C" fn wasm_extern_delete(_item: Option<Box<wasm_extern_t>>) {}
-
```

Comment:
What happened to this function?

Reply:
It's been moved into `externals/mod.rs`, with the other `wasm_extern_*` type and functions.